File: Editor/src/main/java/org/turbomedia/turboedit/editor/panes/player/PlayerPane.java
package org.turbomedia.turboedit.editor.panes.player;

import javafx.geometry.Insets;
import javafx.scene.layout.VBox;

import java.io.IOException;

public class PlayerPane extends VBox {

    public PlayerPane() {
        setSpacing(15);
        setMinWidth(1400);
        setMinHeight(800);
        setPadding(new Insets(15,15,0,15));

        try {
            getChildren().add(new PlayerPlayback());
        } catch (IOException e) {
            throw new RuntimeException(e);
        }

        getChildren().add(new PlayerControls());
    }
}

File: Editor/src/main/kotlin/turboedit/editor/panes/player/PlayerPane.kt
package turboedit.editor.panes.player

import javafx.geometry.Insets
import javafx.scene.layout.VBox
import java.io.IOException

class PlayerPane : VBox() {
    init {
        spacing = 6.0
        minWidth = 1400.0
        minHeight = 800.0
        padding = Insets(15.0)

        try {
            children.add(PlayerPlayback())
        } catch (e: IOException) {
            throw RuntimeException(e)
        }

        children.add(PlayerControls())
    }
}